recipes/r-tfruns/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = '1.3' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-tfruns
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: tfruns_{{ version }}.tar.gz
+  url:
+    - {{ cran_mirror }}/src/contrib/tfruns_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/tfruns/tfruns_{{ version }}.tar.gz
+  sha256: 962aadf373802a4ef1f9a38c25dfa4f6d8bbedf656d0de589953572dc71c2b0a
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+
+  host:
+    - r-base
+    - r-base64enc
+    - r-config
+    - r-jsonlite >=1.2
+    - r-magrittr
+    - r-reticulate
+    - r-rlang
+    - r-rstudioapi >=0.7
+    - r-tidyselect
+    - r-whisker
+    - r-yaml
+
+  run:
+    - r-base
+    - r-base64enc
+    - r-config
+    - r-jsonlite >=1.2
+    - r-magrittr
+    - r-reticulate
+    - r-rlang
+    - r-rstudioapi >=0.7
+    - r-tidyselect
+    - r-whisker
+    - r-yaml
+
+test:
+  commands:
+    - $R -e "library('tfruns')"           # [not win]
+    - "\"%R%\" -e \"library('tfruns')\""  # [win]
+
+about:
+  home: https://github.com/rstudio/tfruns
+  license: Apache 2.0
```

Comment:
Can you add the license_file please

Reply:
Of course. Added. Can you retry circleci build? I can't trigger myself and it seems to have failed only due to an intermittent HTTP error.